### DISCUSSION TITLE
Has the latest version been confirmed to work with the latest VS Code?

### DISCUSSION THREAD

**Discussion by gstorhaug** [pierre3/PlantUmlClassDiagramGenerator#70] [Q&A]
I used CSharp to PlantUML to generate a bunch of Class diagrams last month.
However when I try running it today (2023-03-09), nothing happens (plantuml folder is not even created).  I noticed that VS Code has updated once, and CSharp to PlantUML has been updated twice since I last used it.  There could of course be something other wrong with my system, thus I just hoping for a confirmation that the latest versions work together.

Already generated .puml files are still are displayed correct using  Alt-D (though I realize this has nothing directly to do with CSharp to PlantUML).

**Comment by pierre3** [comment]
I apologize for the delayed response. 
If the command is not functioning properly, please check the "output" section. To view a log of the command's execution, select "CSharp to PlantUML" in the "Output" tab of the terminal panel. If an error occurs, you can review the error details.

![image](https://user-images.githubusercontent.com/1255359/235545117-f8e9a2ef-9e5f-4db0-ae2a-4bd6d2d1c2f2.png)

![image](https://user-images.githubusercontent.com/1255359/235545291-989a1207-45c9-4a5b-af39-8410199ef95a.png)
